.github/workflows/reassurePerformanceTests.yml

```diff
@@ -43,3 +43,14 @@ jobs:
         with:
           DURATION_DEVIATION_PERCENTAGE: 20
           COUNT_DEVIATION: 0
+
+      - name: Get and save graphite string
+        # run only when merged to master
+        if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'main'
+        id: saveGraphiteString
+        uses: ./.github/actions/javascript/getGraphiteString
+
+      - name: Send graphite data
+        # run only when merged to master
+        if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'main'
+        run: echo -e "${{ steps.saveGraphiteString.outputs.GRAPHITE_STRING }}" | nc "$GRAPHITE_SERVER" "$GRAPHITE_PORT"
```

Comment:
"$GRAPHITE_SERVER" "$GRAPHITE_PORT" - will need to be provided and set as github env variables